docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -324,12 +324,23 @@ Release
 
       admin/upload-rpms "${VERSION}"
 
-#. Build and upload the tutorial :ref:`Vagrant box <build-vagrant-box>`.
+#. Copy the tutorial box to the final location:
+   
+   .. note:: Skip this step for a documentation release.
+
+   .. prompt:: bash $
+
+      gsutil cp -a public-read gs://clusterhq-vagrant-buildbot/tutorial/flocker-tutorial-${VERSION}.box gs://clusterhq-vagrant/flocker-tutorial-${VERSION}.box
+
+#. Add the tutorial box to Atlas:
 
    .. note:: Skip this step for a documentation release.
 
-   .. warning:: This step requires ``Vagrant`` and should be performed on your own workstation;
-                **not** on a :doc:`Flocker development machine <vagrant>`.
+   .. prompt:: bash $
+
+      echo http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
```

Comment:
I know it's verified later when we build the tutorial boxes and run the acceptance tests, but:

`curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box`

...might be a useful extra check at this stage to ensure that the box is available at that URL.

```
(test)[vagrant@localhost ~]$ export VERSION=0.3.2

(test)[vagrant@localhost ~]$ curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-${VERSION}.box
HTTP/1.1 200 OK
Expires: Thu, 19 Feb 2015 13:17:00 GMT
Date: Thu, 19 Feb 2015 12:17:00 GMT
Cache-Control: public, max-age=3600
Last-Modified: Wed, 19 Nov 2014 17:11:59 GMT
ETag: "0e71dc22e2b8fa3b874c2928f7bad497"
x-goog-generation: 1416417119309000
x-goog-metageneration: 3
x-goog-stored-content-encoding: identity
x-goog-stored-content-length: 569319802
Content-Type: application/vnd.previewsystems.box
Content-Language: en
x-goog-hash: crc32c=kQeT6Q==
x-goog-hash: md5=DnHcIuK4+juHTCko97rUlw==
x-goog-storage-class: STANDARD
Accept-Ranges: bytes
Vary: Accept-Encoding
Content-Length: 569319802
Server: UploadServer ("Built on Feb 9 2015 19:54:00 (1423540440)")
Alternate-Protocol: 80:quic,p=0.08
```

...as opposed to...

```
(test)[vagrant@localhost ~]$ curl --HEAD http://storage.googleapis.com/clusterhq-vagrant/flocker-tutorial-1.0.0.box
HTTP/1.1 404 Not Found
Content-Type: application/xml; charset=UTF-8
Content-Length: 127
Date: Thu, 19 Feb 2015 12:21:26 GMT
Expires: Thu, 19 Feb 2015 12:21:26 GMT
Cache-Control: private, max-age=0
Server: UploadServer ("Built on Feb 9 2015 19:54:00 (1423540440)")
Alternate-Protocol: 80:quic,p=0.08

```